contracts/agToken/BaseAgTokenSideChain.sol

```diff
@@ -84,19 +84,18 @@ contract BaseAgTokenSideChain is IAgToken, ERC20PermitUpgradeable {
     /// @param name_ Name of the token
     /// @param symbol_ Symbol of the token
     /// @param _treasury Reference to the `Treasury` contract associated to this agToken implementation
-    /// @dev By default, agTokens are ERC-20 tokens with 18 decimals
     function _initializeBase(
         string memory name_,
         string memory symbol_,
         address _treasury
-    ) internal virtual {
+    ) internal virtual initializer {
```

Comment:
don't need `intializer` this one as it is always called by a function that already have it